### PR TITLE
fix(FEC-12717): add timeline dependency to ima plugin

### PR DESCRIPTION
**Pull request by amirch1**
### Description of the Changes

Please add a detailed description of the change, weather it's an enhancement or a bugfix.
If the PR is related to an open issue please link to it.

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

**Comment by SivanA-Kaltura** [comment]
@amirch1 oh, are you merging from a fork ? that's why it has the kaltura: prefix then

**Review by SivanA-Kaltura — APPROVED** [review]

**Review by giladna — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -114,6 +114,12 @@
   "publishConfig": {
     "access": "public"
   },
+  "kaltura": {
+    "name": "ima",
+    "dependencies": {
+      "playkit-timeline": "3.0.0"
```
Comment (SivanA-Kaltura) [inline package.json:120]: @amirch1 
should it be timeline or playkit-timeline ? assuming it uses "name", it's timeline 
https://github.com/kaltura/playkit-js-timeline/blob/8eb0755ca49dbca6fb99e5fb2c40e846a2040a5d/package.json#L114

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (amirch1) [reply package.json:120]: @SivanA-Kaltura I used https://github.com/kaltura/playkit-js-ivq/blob/master/package.json#L65 as reference. This works fine for IVQ. The name is not the plugin name but the bundle name

**Review by SivanA-Kaltura — APPROVED** [review]

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]
@amirch1 @giladna we need to discuss this PR

**Pull request merged**